nginx.conf

```diff
@@ -57,4 +58,85 @@ http {
             proxy_pass http://minio;
         }
     }
+
+    # Currently, uncommenting even this breaks the nginx proxying for Minio
+    # a few requests go through to the science archive and then nginx -T says:
+    #nginx: [emerg] host not found in upstream "minio1:9000" in /etc/nginx/nginx.conf:26
+    #nginx: configuration file /etc/nginx/nginx.conf test failed
+
+
+    # Need to get this working, then add the proxy endpoints for zip files
+    server {
+        listen 9500;
```

Comment:
Interesting, I swear I tried this to no success! Maybe it was the changing of the archive api to run on port 9501?

Reply:
yeah that might do it... but im not sure how it wouldve made minio unavailable, i think just the archiveapi wouldve been unavailable then.